src/notte/llms/engine.py

```diff
@@ -50,20 +55,37 @@ def structured_completion(
         messages: list[AllMessageValues],
         response_format: type[TResponseFormat],
         model: str | None = None,
+        retries: int = 3,
```

Comment:
can this be set to 0 | None by default ? And override this parameter in a config somwhere in `AgentConfig` or `EnvConfig` ?